test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-result-dir.yml

```diff
@@ -0,0 +1,7 @@
+gcloud:
+  app: ./src/test/kotlin/ftl/fixtures/tmp/apk/app-debug.apk
```

Comment:
Is this used anywhere? I think we inlined the example

Reply:
It's only used for my internal tests of this issue. I removed it from repository.